<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>818</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Burn-In Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 170, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="tempSP_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TempSP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp1_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp1</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp2_label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp2</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp3_label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp3</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentSP_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CurrentSP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startB">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start Burn-In</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="comMessages">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>651</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="v11_display">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="v21_display">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="v31_display">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="v12_display">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="v22_display">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="v32_display">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="v12_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V12</string>
    </property>
   </widget>
   <widget class="QLabel" name="v22_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V22</string>
    </property>
   </widget>
   <widget class="QLabel" name="v32_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V32</string>
    </property>
   </widget>
   <widget class="QLabel" name="v11_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V11</string>
    </property>
   </widget>
   <widget class="QLabel" name="v21_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V21</string>
    </property>
   </widget>
   <widget class="QLabel" name="v31_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V31</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Time Elapsed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="elapsedTimeBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="testContactB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string>Test Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetDeviceB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>630</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Reset Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filenameBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>491</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected USB</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="opaqueResize">
     <bool>false</bool>
    </property>
    <property name="childrenCollapsible">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="currentToggle">
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tempToggle">
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="temp1Displa"/>
    <widget class="QLCDNumber" name="temp2_displa"/>
    <widget class="QLCDNumber" name="temp3_displa"/>
   </widget>
   <widget class="QRadioButton" name="heating1indicator">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="heating2indicator">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>190</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="heating3indicator">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="statusDisplay">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QLabel" name="v31_label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>260</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>I31</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="i11_display">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="v11_label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>I11</string>
    </property>
   </widget>
   <widget class="QLabel" name="v21_label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>I21</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="i32_display">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="v32_label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>370</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>I32</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="i12_display">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="i31_display">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="v22_label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>I22</string>
    </property>
   </widget>
   <widget class="QLabel" name="v12_label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>I12</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="i21_display">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="i22_display">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOpen_Settings"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionOpen_Settings">
   <property name="text">
    <string>Open Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>connectB</tabstop>
  <tabstop>startB</tabstop>
  <tabstop>currentToggle</tabstop>
  <tabstop>tempToggle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
